frontend/src/authentication/components/AuthGuard.js

```diff
@@ -15,6 +16,13 @@ export const AuthGuard = (props) => {
       setRequestedLocation(location.pathname);
     }
 
+    if (
+      !sessionStorage.getItem('window-location') &&
+      location.pathname !== '/'
```

Comment:
Checking this explicitly so that we do not set the '/' which will happen in case a user logsout and then pastes the URL in the browser and again tries to logback with the new URL . As the user logged out the redirect URL is the base URL and then this gets stored in the window-location which should not be the case. We only want to store window location path pointing to specific path